src/sql/error_message.py

```diff
@@ -0,0 +1,36 @@
+import sqlglot
+
+
+def detail(msg, query=None):
+    msg = str(msg)
+    return_msg = "\nLooks like there is a syntax error " "in your query\n"
```

Comment:
```suggestion
    return_msg = "\nLooks like there is a syntax error " " in your query\n"
```

Add a space between `" "` and `in`

Reply:
These quotes are added when formatting using `pkgmt`. The output looks fine in notebook. Do you see any errors?
@tonykploomber

Reply:
I see, then no worries